tezos/new_context/src/tezedge_context.rs

```diff
@@ -661,21 +699,35 @@ impl ShellContextApi for TezedgeContext {
         // Entries to be inserted are obtained from the commit call and written here
         let date: u64 = date.try_into()?;
         let mut repository = self.index.repository.write()?;
-        let (commit_hash_id, batch, referenced_older_entries) = self.tree.prepare_commit(
+
+        let PostCommitData {
+            commit_hash_id,
+            batch,
+            reused,
+            serialize_stats,
+        } = self.tree.prepare_commit(
             date,
             author,
             message,
             self.parent_commit_hash,
             &mut *repository,
             true,
         )?;
+
+        // TODO: Store serialize_stats somewhere
```

Comment:
For this one open a Jira issue so that it doesn't get forgotten, and update the comment to `TODO - TE-NNN: ...`